moldesign/interfaces/biopython_interface.py

```diff
@@ -24,6 +24,12 @@
 from moldesign.helpers.pdb import BioAssembly
 
 
+def exports(o):
```

Comment:
Thanks, this is a good question (and definitely confusing). The `@exports` decorator adds a function's name to the [module's `__all__` variable](http://code.activestate.com/recipes/576993-public-decorator-adds-an-item-to-__all__/). It marks the functions that we want to expose in a given subpackage's API.

Basically,
```python
@exports
def my_function(): [...]
```

is completely equivalent to
```python
__all__ = ['my_function', ...]

def myfunction(): [...]
```

but much more readable (in my opinion)